scripts/windows/build-dotnet-trace.bat

```diff
@@ -0,0 +1,62 @@
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Linq.Expressions.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Memory.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Private.CoreLib.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Private.Uri.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Runtime.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Runtime.Extensions.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Runtime.InteropServices.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Runtime.InteropServices.RuntimeInformation.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Text.RegularExpressions.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Threading.Channels.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Threading.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Threading.Overlapped.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Threading.Tasks.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Threading.Thread.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Threading.ThreadPool.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.IO.Pipes.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Net.Sockets.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Net.Primitives.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Security.Principal.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.IO.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Security.Cryptography.Algorithms.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Security.Cryptography.Primitives.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\libSystem.Security.Cryptography.Native.OpenSsl.so"
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Runtime.CompilerServices.Unsafe.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Diagnostics.TraceSource.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Reflection.Emit.ILGeneration.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Reflection.Emit.Lightweight.dll",
+            ".\dep\dotnet\shared\Microsoft.NETCore.App\6.0.0\System.Reflection.Primitives.dll",
+            ) DO COPY %%i .\app\gprofiler\resources\dotnet\shared
+ECHO D | XCOPY .\dep\dotnet\tools .\app\gprofiler\resources\dotnet\tools /E
+COPY .\dep\dotnet\host\fxr\6.0.0\hostfxr.dll .\app\gprofiler\resources\dotnet\tools\.store\dotnet-trace\6.0.351802\dotnet-trace\6.0.351802\tools\netcoreapp3.1\any\
```

Comment:
`6.0.351802` is probably likely to change, it seems like a minor/patch version, I suppose when it'll change it will break the build, no?

Reply:
I added the version argument, so it should work well in the future too